frontend-ui/src/components/ScheduleEditor.vue

```diff
@@ -508,6 +508,28 @@
         </p>
       </div>
 
+      <!-- Similar Recipes Information -->
+      <div v-if="loadingSimilarRecipes" class="mb-4">
+        <v-alert type="info" variant="tonal" density="compact">
+          <template #prepend>
+            <v-progress-circular indeterminate size="16" />
+          </template>
+          Checking for similar recipes...
+        </v-alert>
+      </div>
+
+      <div v-else-if="similarRecipesCount !== null && similarRecipesCount > 0" class="mb-4">
+        <v-alert type="warning" variant="tonal" density="compact">
+          <template #prepend>
+            <v-icon>mdi-information</v-icon>
+          </template>
+          There are {{ similarRecipesCount }} similar recipe{{
```

Comment:
`There are {{ similarRecipesCount }} similar recipe{{` => `There are {{ similarRecipesCount }} recipe(s) similar {{` 

I feel like this formulation is more correct (with plural by default), WDYT?